h2/src/main/org/h2/mvstore/Chunk.java

```diff
@@ -341,6 +360,10 @@ protected void dump(StringBuilder buff) {
         }
     }
 
+    public String getHeader() {
+        return new String(getHeaderBytes(), StandardCharsets.ISO_8859_1);
```

Comment:
why not UTF8 ?

Reply:
It's just been like that from day 1. It should make zero difference since we are talking lower ASCII characters only.

Reply:
`StandardCharsets.ISO_8859_1` is the fastest encoding in JVMs with compact strings.